lib/ansible/utils/module_docs_fragments/nxos.py

```diff
@@ -73,6 +73,12 @@ class ModuleDocFragment(object):
     required: false
     default: no
     choices: ['yes', 'no']
+  timeout:
+    description:
+      - Specifies idle timeout for the connection.  NX-API can be slow to 
```

Comment:
Can you please add the units, e.g.
"Specifies idle timeout for the connection, in seconds."